### DISCUSSION TITLE
Flux bootstrap Bitbucket

### DISCUSSION THREAD

**Discussion by AmitP30** [fluxcd/flux2#1170] [Q&A]
Hello,

We would like to bootstrap bitbucket repository? Is that feature available right now?
Error: ✗ failed to create repository, error: POST https://bitbucket.bre.mcd.com/api/v3/orgs/bitbucket/repos: 401  []

**Comment by stealthybox** [comment]
401 is an authentication error.
What command are you using?
We haven't released a bootstrap for bitbucket yet -- only GitHub and GitLab

**Reply by gerardgorrion** [reply]
There're expected to add bootstrap for bitbucket?

**Comment by stefanprodan** [comment]
We don't have native support for Bitbucket but we do support any Git server with `flux bootstrap git`, please see the docs here: https://toolkit.fluxcd.io/guides/installation/#generic-git-server

**Comment by stefanprodan** [comment]
In Flux [0.24.0](https://github.com/fluxcd/flux2/releases/tag/v0.24.0) we've added support for **BitBucket Server and Data Center** to bootstrap, docs here: https://fluxcd.io/docs/installation/#bitbucket-server-and-data-center
